docs/Using-Fleet/MDM-commands.md

```diff
@@ -71,5 +71,18 @@ Fleet UI:
 
 2. Look at the on-screen information.
 
+## List of recent commands
+
+You can view the list of recently executed commands using "fleetctl":
+
+1. Run `fleetctl get mdm-commands`
+2. View the list of recently executed commands, most recent first, along with the timestamp, targeted hostname, command type, execution status and command ID.
+
+The command ID can be used to view command results as documented in [Step 4 of the previous section](#step-4-view-the-commands-results). The possible status values are:
+* Pending: the command has yet to be executed on the host.
```

Comment:
```suggestion
* Pending: the command has yet to run on the host. The host will run the command the next time it comes online.
```

Minor nit. We're using "run" language elsewhere in the docs. I think we should be consistent.

Also I added a bit of context to let the user know when the command will run.